cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py

```diff
@@ -42,6 +45,41 @@ def _is_segm_id_test(self, param, name):
         return param in PARAMS_NAMES_TO_OMIT and (
             name in NET_NAMES_TO_OMIT or name in SUBNET_NAMES_TO_OMIT)
 
+    def get_attribute_val(self, attr_list, resource_name, parameter):
+        temp = {}
+        temp[resource_name] = {}
+        temp[resource_name][parameter] = getattr(attr_list, parameter)
+        temp[resource_name]["name"] = getattr(attr_list, "name").lower()
+        return temp[resource_name][parameter]
+
+    def validate_tenant_resource_param_in_dst(self, src_list, dst_list,
+                                              resource_name='tenant',
+                                              parameter='name'):
+        if not src_list:
+            self.skipTest(
+                'Nothing to migrate - source resources list is empty')
+        name_attr = 'name'
+        for i in src_list:
+            for j in dst_list:
+                if self.get_attribute_val(i, resource_name, name_attr) != \
+                        self.get_attribute_val(j, resource_name, name_attr):
+                    continue
+                if getattr(i, parameter, None):
+                    if self.get_attribute_val(i, resource_name, parameter) != \
+                           self.get_attribute_val(j, resource_name, parameter):
+                        msg = 'Parameter {param} for resource {res} with ' \
+                              'name {name} are different src: {r1}, dst: {r2}'
+                        self.fail(msg.format(
+                            param=parameter, res=resource_name,
+                            name=getattr(i, name_attr),
+                            r1=getattr(i, parameter),
+                            r2=getattr(j, parameter)))
+                break
+            else:
+                msg = 'Resource {res} with name {r_name} was not found on dst'
+                self.fail(msg.format(res=resource_name,
```

Comment:
I think you can fix this one, and then create task for other validate_* functions pointing how it was done here.